Sources/ApolloCodegenLib/Frontend/dist/ApolloCodegenFrontend.bundle.js

```diff
@@ -1,2 +1 @@
-//# sourceMappingURL=ApolloCodegenFrontend.bundle.js.map
```

Comment:
Why is this no longer necessary? And why did the bundle map get deleted?

Reply:
It was never necessary, this is just cleanup of legacy. I changed the rollup config that creates the bundled js to remove the source mapping. As far as I can tell. that is only used for TypeScript libraries, and ours is pure JS.